circuits/circom/isDeactivatedKey.circom

```diff
@@ -0,0 +1,54 @@
+pragma circom 2.0.0;
+include "./trees/incrementalQuinTree.circom";
+include "../node_modules/circomlib/circuits/comparators.circom";
+include "./poseidon/poseidonHashT3.circom";
+
+template IsDeactivatedKey(levels) {
+    var LEAVES_PER_NODE = 5;
+    var LEAVES_PER_PATH_LEVEL = LEAVES_PER_NODE - 1;
+
+    signal input root;
+    signal input key[2];
+
+    // Ciphertext of the encrypted key status
+    signal input c1;
```

Comment:
what are c1 and c2's definition here?

Reply:
c1 and c2 are parts of the ElGamal ciphertext of the key deactivation status (success/failure).